book/first-figure.ipynb

```diff
@@ -0,0 +1,512 @@
+    }
+   },
+   "cell_type": "markdown",
+   "id": "bc713bc7-6c8b-4709-b683-e7908b4a91ef",
+   "metadata": {},
+   "source": [
+    "#### Coloring and add map element (frame and ticks)! \n",
+    "1. Axis labels `WSne`: If an upper-case letter (W, S, N, E) is passed, the axis is plotted with tick marks and annotations. The lower-case version (w, s, n, e) plots the axis only with tick marks. To only plot the axis pass (l, b, t, r).\n",
+    "\n",
+    "2. annotations `a`: tick labels\n",
+    "3. frame `f`: ticks\n",
+    "4. grid `g`: grid line in the figure. \n",
+    "![image.png](attachment:ecb6ec76-f78f-4bac-9556-f6280ca7cd43.png)\n",
+    "\n",
+    "> More detail for frame https://www.pygmt.org/dev/tutorials/basics/frames.html\n",
+    "\n",
+    "> More color https://docs.generic-mapping-tools.org/dev/_images/GMT_RGBchart.png "
```

Comment:
Try to link to a specific version if possible (the 'dev' version can change in the future).

```suggestion
    "> More detail for frame https://www.pygmt.org/v0.13.0/tutorials/basics/frames.html\n",
    "\n",
    "> More colors https://docs.generic-mapping-tools.org/6.5/gmtcolors.html#list-of-colors"
```